e2e-tests/kubernetes-headlamp-ci.yaml

```diff
@@ -6,7 +6,7 @@ metadata:
   namespace: kube-system
 spec:
   accessModes:
-    - ReadWriteMany
+    - ReadWriteOnce
```

Comment:
What is this change for? I don't see it mentioned in the git message.

Reply:
I noticed locally that this fixed an issue where the headlamp deployment timed out, otherwise CI would fail here